sympy/physics/units/tests/test_dimensions.py

```diff
@@ -100,8 +100,7 @@ def test_Dimension_mul_div_exp():
     assert dimsys_SI.get_dimensional_dependencies(length ** -1) == {"length": -1}
     assert dimsys_SI.get_dimensional_dependencies(velo ** -1.5) == {"length": -1.5, "time": 1.5}
 
-    length_a = length**"a"
-    assert dimsys_SI.get_dimensional_dependencies(length_a) == {"length": Symbol("a")}
```

Comment:
why is this being removed?

Reply:
I've changed the behavior to require exponents to be dimensionless so I've updated the test to reflect this. As far as I understand, an exponent should be dimensionless.

Reply:
Yes, but it can still be a symbol. The symbol may still me dimensionless.

Reply:
Okay, I see, thanks for the clarification. I've updated the behavior to allow a Symbol, in addition to a Number, as an exponent. A Dimension as an exponent raises an excpetion. I restored the original test that you highlighted to make sure a Symbol works as an exponent.

Reply:
Fixed a regression I introduced that didn't allow a dimensionless expression as an exponent. This has been fixed and I added an additional test case for this regression.